components/ide/code/BUILD.yaml

```
@@ -7,7 +7,10 @@ packages:
       - "bin/*"
     argdeps:
       - imageRepoBase
+      - gitpodDomain
     config:
       dockerfile: leeway.Dockerfile
       image:
         - ${imageRepoBase}/ide/code:${version}
+      buildArgs:
+        OPENVSX_URL: https://open-vsx.${gitpodDomain}
```

if we made this an explicit variable, we could set a workspace default and only produce a build when absolutely necessary. 

```suggestion
        OPENVSX_URL: ${OPENVSX_URL}
```